src/app/components/core/control-dispatcher/add-control-acl/add-control-acl.component.ts

```diff
@@ -42,7 +42,8 @@ export class AddControlAclComponent implements OnInit {
     private dialogService: DialogService,
     private docService: DocService,
     private servicesApiService: ServicesApiService,
-    public sharedService: SharedService) { }
+    public sharedService: SharedService,
```

Comment:
`sharedService` can be removed from here. There is no use of `SharedService` in this file now.